<launch>
    <!-- Turtlesim Node-->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="log"/>

    <node name="turtle_controller" pkg="srd_exercises" type="turtle_controller.py" respawn="false" output="screen" >
      <param name="turtlename" type="string" value="turtle1"/>
      <param name="sat_v" type="double" value="4.0"/>
      <param name="sat_w" type="double" value="4.0"/>
    </node>
    <node name="turtle_random_cmd_limit" pkg="srd_exercises" type="turtle_random_cmd_limit.py" respawn="false" output="screen" >
      <param name="turtlename" type="string" value="turtle1"/>
      <param name="y0" type="double" value="4.0"/>
    </node>

</launch>
